latest/index.html

```diff
@@ -321,6 +321,12 @@ <h3>Service Manifest</h3>
           <dt><code>defaultTypes</code></dt>
           <dd>An array of <a>types</a> which are considered sensible default choices as types supplied in reconciliation queries. For services which do not rely on types, this MAY contain a single type with a generic name making it clear that all entities in the
 database are instances of this type.<dd>
+          <dt><code>documentation</code></dt>
+          <dd>An optional URL with human-readable documentation about the service, for instance giving more information about the data it exposes;</dd>
+          <dt><code>logo</code></dt>
+          <dd>An optional URL of a square image which can be used as the service's logo;</dd>
+          <dt><code>serviceVersion</code></dt>
+          <dt>An optional string which describes the version of the software exposing this service. This is not to be confused with <code>versions</code> which is about the versions of the protocol supported by the service;</dt>
```

Comment:
```suggestion
          <dt>An optional string which describes the version of the software exposing this service. This is not to be confused with <code>versions</code> which is about the versions of the reconciliation api supported by the service;</dt>
```

Reply:
I think saying `protocol` felt slightly out of place here.  If I'm wrong, that's fine.

Reply:
Fine with me!